portfolio/src/main/webapp/index.html

```diff
@@ -59,8 +59,12 @@ <h2>Hidden Talent? </h2>
       <img src="/myImages/tournament.jpeg" alt="taekwondo" class="tournament"><br>
 
       <p>Click here to get a random quote from my favorite TV shows or fact about me:</p>
-      <button onclick="addRandomGreeting()">Click</button>
-      <div id="greeting-container"></div>
+      <button onclick="addRandomFacts()">Click</button>
+      <div id="container"></div><br>
+
+      <h3> Fetch message: </h3>  
```

Comment:
nit: remove the leading and trailing white space

Reply:
Noted!